### DISCUSSION TITLE
posts/2022-my/%E5%B7%A5%E4%BD%9C/%E8%80%83%E8%AF%95%E5%BE%AE%E6%9C%8D%E5%8A%A1-%E6%94%B9%E5%8D%B7%E7%AE%A1%E7%90%86-%E6%8E%A5%E5%8F%A3%E4%BC%98%E5%8C%96

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#287] [Announcements]
# posts/2022-my/%E5%B7%A5%E4%BD%9C/%E8%80%83%E8%AF%95%E5%BE%AE%E6%9C%8D%E5%8A%A1-%E6%94%B9%E5%8D%B7%E7%AE%A1%E7%90%86-%E6%8E%A5%E5%8F%A3%E4%BC%98%E5%8C%96

人若无名，专心练剑

http://www.iocaop.com/posts/2022-my/%E5%B7%A5%E4%BD%9C/%E8%80%83%E8%AF%95%E5%BE%AE%E6%9C%8D%E5%8A%A1-%E6%94%B9%E5%8D%B7%E7%AE%A1%E7%90%86-%E6%8E%A5%E5%8F%A3%E4%BC%98%E5%8C%96.html

**Comment by yi1108** [comment]
11111
